Sources/StreamChat/Workers/CurrentUserUpdater.swift

```diff
@@ -119,16 +119,30 @@ class CurrentUserUpdater: Worker {
     ///     - currentUserId: The current user identifier.
     ///     - completion: Called when request is successfully completed, or with error.
     func fetchDevices(currentUserId: UserId, completion: ((Error?) -> Void)? = nil) {
+        fetchDevices(currentUserId: currentUserId, includeResponse: false, completion: { completion?($0.error) })
+    }
+    
+    private func fetchDevices(currentUserId: UserId, includeResponse: Bool, completion: ((Result<[Device], Error>) -> Void)? = nil) {
```

Comment:
Why don't we always include the device response? Since this is a new major version, we can change this. IMO we should change it. From now on, all responses should return the actual payload 🤔 
```suggestion
    private func fetchDevices(currentUserId: UserId, completion: ((Result<[Device], Error>) -> Void)? = nil) {
```

Reply:
+1 on this

Reply:
I can change it on the updater level, but not breaking the existing public interface since it is used by controller

Reply:
Changed it on the updater level. Done.